### PR TITLE
moved openapi-types to be a direct dependency to satisfy type definit…

### PR DESCRIPTION
**Pull request by zelein**
Moved the openapi-types package to be a direct dependency. This will allow the types to be resolved in the included `index.d.ts` file.

Addresses #117

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8056,8 +8056,7 @@
     "openapi-types": {
       "version": "1.3.5",
       "resolved": "https://registry.npmjs.org/openapi-types/-/openapi-types-1.3.5.tgz",
-      "integrity": "sha512-11oi4zYorsgvg5yBarZplAqbpev5HkuVNPlZaPTknPDzAynq+lnJdXAmruGWP0s+dNYZS7bjM+xrTpJw7184Fg==",
-      "dev": true
```
Comment (zelein) [inline package-lock.json]: For some reason the "dev" property did not get removed when I moved the dependencies. I tried a few different things and the `openapi-types` module wouldn't even get removed when I ran `npm remove openapi-types`.

I manually edited this file to remove this flag. I primarily use yarn, so am no longer in touch with the nuances of npm.

**Comment by ffflorian** [comment]
Hi @zelein, great to see your PR, I also need this for https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36165. So hopefully this gets merged soon :slightly_smiling_face:

**Comment by JamesMessinger** [comment]
Thanks!  Good catch

**Comment by ffflorian** [comment]
:tada:

**Pull request merged**